README.md

```diff
@@ -27,6 +27,8 @@ Po zainstalowaniu powyższego oprogramowania pozostaje nam ściągnąć projekt
 I ostatnim krokiem jest wykonanie w katalogu projektu:
 ```vagrant up```
 
+Inną możliwością jest skorzystanie z programu **XAMPP** lub podobnego, oferującego serwer ```Apache``` i postępowanie zgodnie z niniejszą instrukcją: https://drive.google.com/file/d/0B_0lVIhwL9LWMjV1eGJyTk0yMTQ/view?pref=2&pli=1
```

Comment:
Wolny wybór, opisane pięknie na Linuxie i Vagrancie, ale jak ktoś bardzo chce bez tego to niech też ma instrukcję ;)